.gitignore

```diff
@@ -28,7 +20,6 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
-# project.xcworkspace
```

Comment:
this one should be added

Reply:
What do you mean by this? It's a comment that I removed. The file is more in line with the React Native default. Any specific requirements for this project?